src/interp/binary-reader-interp.cc

```diff
@@ -1229,29 +1231,31 @@ Result BinaryReaderInterp::OnLocalTeeExpr(Index local_index) {
 
 Result BinaryReaderInterp::OnLoadExpr(Opcode opcode,
                                       Address align_log2,
-                                      Address offset) {
-  CHECK_RESULT(validator_.OnLoad(loc, opcode, GetAlignment(align_log2)));
-  istream_.Emit(opcode, kMemoryIndex0, offset);
+                                      Address offset,
+                                      Index memidx = 0) {
```

Comment:
Is the `= 0` needed here and below?   i.e. Are there callsites that don't pass memidx?